STYLE-GUIDE.md

```diff
@@ -1,19 +1,19 @@
 # OpenRC Style Guide
 
-This is the openrc style manual.  It governs the coding style of all code
-in this repository.  Follow it.  Contact openrc@gentoo.org for any questions
+This is the openrc style manual. It governs the coding style of all code
```

Comment:
I didn't know there is another styling except for a single space. I have always considered two spaces to be typos.